SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/Storage.cs

```diff
@@ -52,7 +53,7 @@ public Storage() : this(DefaultAdapter())
         internal Storage(IStorageAdapter adapter)
         {
             _storageAdapter = adapter;
-            _queue.Add(new Task(() => InitializeDatabaseAsync().Wait()));
```

Comment:
Like @MatkovIvan mentioned in a conversation we had, just using `.GetAwaiter().GetResult()`  would remove the need for an extension utils, please switch to that and remove the additional extension.